files/dashboards/openshift-logging-dashboard.json

```diff
@@ -1708,18 +1798,233 @@
         "align": false,
         "alignLevel": null
       }
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": null,
+      "fieldConfig": {
+        "defaults": {
+          "custom": {},
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          }
+        },
+        "overrides": []
+      },
+      "fill": 1,
+      "fillGradient": 0,
+      "gridPos": {
+        "h": 7,
+        "w": 8,
+        "x": 16,
+        "y": 49
+      },
+      "hiddenSeries": false,
+      "id": 41,
+      "legend": {
+        "avg": false,
+        "current": false,
+        "max": false,
+        "min": false,
+        "show": true,
+        "total": false,
+        "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "nullPointMode": "null",
+      "percentage": false,
+      "pluginVersion": "7.1.1",
+      "pointradius": 2,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "topk(10, increase(fluentd_input_status_total_bytes_logged[15m]) > 0)",
```

Comment:
@ajaygupta978 I know that we agreed that rate and increase are similar but I think that changing to rate function will make things more understood ... also ant particular reason for 15m ... maybe reduce that number so that the metric will update more quickly ... how about 5 minutes ?

Reply:
@ajaygupta978 in any event it will be good to normalize the information presented to users so that it is per seconds and not related to the period of metric collection (i.e. 15m or any other value)

Reply:
Yeah agree even I noticed in the code other charts are using 5m and rate function make more sense because its normalises.